### ISSUE TITLE
Keras Conv1D Keras for 2D data

### ISSUE THREAD

**Issue by anilmaddala** [keras-team/keras#8042]
I am new to Keras and trying to implement a time series signal classifier. The signal is in the form of csv [signal_class, time_stamp, X, Y]

I am able to load the data succesfully and plot the training data.

The window size is 60 and there are only 2 possible classes. What should my network architecture be? How do I reshape the data for Conv1D input?

I am currently following [Human Activity Recognition based on accelerometer data](https://github.com/aqibsaeed/Human-Activity-Recognition-using-CNN/blob/master/Activity%20Detection.ipynb) and trying to eliminate Z axis. Also this sample code is in Tensorflow and I am trying to port to Keras.

I am using Tensorflow backend. Any help is appreciated. Thanks.

**Comment by techfun77** [comment]
Why You want to use Conv1D?? Maybe better is to use LTSM nets. Check that [Deep Spying](https://arxiv.org/pdf/1512.05616.pdf).
